server/base/src/main/java/org/apache/accumulo/server/metadata/ConditionalTabletMutatorImpl.java

```diff
@@ -251,6 +254,9 @@ public Ample.ConditionalTabletMutator requireSame(TabletMetadata tabletMetadata,
   public void submit(Ample.RejectionHandler rejectionCheck) {
     Preconditions.checkState(updatesEnabled, "Cannot make updates after calling mutate.");
     Preconditions.checkState(sawOperationRequirement, "No operation requirements were seen");
+    if (!prevEndRowSupplied && !requireAbsentTabletCalled) {
+      requireAbsentTablet();
+    }
```

Comment:
Could add the end row condition check here if require absent was not called.

```suggestion
    if (!requireAbsentTabletCalled) {
      // TODO add end row condition check
    }
```

If doing that then could remove the mutateTablet method that takes an endRow